ui/css/intermittent-failures.css

```diff
@@ -70,3 +70,28 @@ ul {
   list-style-type: none;
   text-align: left;
 }
+
+/* Graphic overriding style */
```

Comment:
What does graphic mean? Could you be more specific about what it's overriding?

Reply:
Half a month later I don't remember what it does. So yeah, I should have been more specific haha